### PR TITLE
Added credentials forms for more clouds.

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- open the deployment flow with `/:flags:/blues/`
- notice the new cloud options
- clicking on a new option should show the form for that cloud

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4799/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4813/
Test PASSed.

**Comment by anthonydillon** [comment]
Could you please optimise the new aws.svg. It can be reduced by half using SVGO.

File: jujugui/static/gui/src/app/components/deployment/add-credentials/add-credentials.js [hunk 1]
```diff
@@ -157,6 +130,128 @@ YUI.add('deployment-add-credentials', function() {
...
+                  placeholder="us-central1"
+                  required="required"
+                  type="text"
+                  ref="templateRegion" />
+              </div>
+              <div className="deployment-panel__notice six-col last-col">
+                <juju.components.SvgIcon
+                  name="general-action-blue"
+                  size="16" />
+                Credentials are store securely on our servers and we'll notify
```
Comment (makyo) [inline jujugui/static/gui/src/app/components/deployment/add-credentials/add-credentials.js]: "Credentials are stored..."

File: jujugui/static/gui/src/app/components/deployment/add-credentials/test-add-credentials.js [hunk 2]
```diff
@@ -55,66 +84,300 @@ describe('DeploymentAddCredentials', function() {
...
+                placeholder="us-central1"
+                required="required"
+                type="text"
+                ref="templateRegion" />
+            </div>
+            <div className="deployment-panel__notice six-col last-col">
+              <juju.components.SvgIcon
+                name="general-action-blue"
+                size="16" />
+              Credentials are store securely on our servers and we'll notify
```
Comment (makyo) [inline jujugui/static/gui/src/app/components/deployment/add-credentials/test-add-credentials.js]: 'stored' here too

**Comment by makyo** [comment]
:+1: QA :ok:

**Comment by hatched** [comment]
@anthonydillon the svg's are automatically optimized in the build process so no need to pre-optimize

**Comment by jaycee** [comment]
:+1:

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4835/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4837/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4840/
Test PASSed.

**Comment by huwshimi** [comment]
Thanks everyone. @anthonydillon as @hatched mentioned the svgs are optimised by our build process (from memory it uses SVGO).

**Comment by huwshimi** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**